packages/electrode-archetype-react-app-dev/config/webpack/util/logger.js

```diff
@@ -0,0 +1,61 @@
+const winston = require("winston");
+const bunyan = require("bunyan");
+
+const archetype = require("electrode-archetype-react-app/config/archetype");
+const enableWinstonLog = archetype.webpack.enableWinstonLog;
+
+const winstonLogger = new (winston.Logger)({
+  exceptionHandlers: [
+    new (winston.transports.File)({
+      filename: "exceptions.log"
+    })
+  ],
+  transports: [
+    new (winston.transports.Console)({
+      colorize: true,
+      level: 'info',
+      prettyPrint: true
+    }),
+    new (winston.transports.File)({
+      name: 'error-file',
+      filename: 'winston-error.log',
+      level: 'error'
+    }),
+    new (winston.transports.File)({
+      name: 'warn-file',
+      filename: 'winston-warn.log',
+      level: 'warn'
+    }),
+    new (winston.transports.File)({
+      name: 'info-file',
+      filename: 'winston-info.log',
+      level: 'info'
+    }),
+    new (winston.transports.File)({
+      name: 'verbose-file',
+      filename: 'winston-verbose.log',
+      level: 'verbose'
```

Comment:
A single file, with all logs.

Name should be something more generally related to electrode archetype, not something that relates to winston or whatever logger is used.